nuxt.config.js

```diff
@@ -90,7 +90,6 @@ export default {
 		"@nuxtjs/pwa",
 		// Doc: https://github.com/nuxt-community/dotenv-module
 		"@nuxtjs/dotenv",
-		"@nuxtjs/style-resources",
```

Comment:
Shall package-lock.json be adjusted too? 

BTW, could you add a "ready for another review" comment when all comments are addressed/tested and ready for another look? It saves reviewers time to track new commits. Thanks.

Reply:
Okay, thank you for the tip